torch_em/model/probabilistic_unet.py

```diff
@@ -196,17 +194,18 @@ class Fcomb(nn.Module):
     and output of the UNet (the feature map) by concatenating them along their channel axis.
     """
     def __init__(
-                self,
-                num_filters,
-                latent_dim,
-                num_output_channels,
-                num_classes,
-                no_convs_fcomb,
-                initializers,
-                use_tile=True
-            ):
-
-        super(Fcomb, self).__init__()
+        self,
+        num_filters,
+        latent_dim,
+        num_output_channels,
+        num_classes,
+        no_convs_fcomb,
+        initializers,
+        use_tile=True,
+        device='cpu'
```

Comment:
I would rather set this to `None`, and if it's `None` set it to cpu or gpu depending on `torch.cuda.is_available`.

Reply:
True, this makes sense. Thanks for mentioning. It's fixed now!